crm_action/__openerp__.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
-# © 2015-2016 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# Copyright 2015-2016 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# Copyright 2017 Tecnativa - Vicent Cubells
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'CRM Action',
-    'version': '8.0.1.2.0',
-    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'version': '9.0.1.0.0',
+    'author': 'Savoir-faire Linux, '
+              'Tecnativa, '
```

Comment:
I know that @pedrobaeza disagrees, but porting does not grant authorship/copyright.